packages/pipeline-editor/src/PipelineSubmissionDialog.tsx

```diff
@@ -42,23 +42,31 @@ export class PipelineSubmissionDialog extends Widget
       ) as HTMLInputElement).value,
       runtime_config: (document.getElementById(
         'runtime_config'
-      ) as HTMLInputElement).value
+      ) as HTMLInputElement).value,
+      filetype: document.getElementById('pipeline_filetype') as HTMLInputElement
```

Comment:
I think you need `.value` here but also you may need to deal with the case this field doesn't exist